src/aiida/cmdline/commands/cmd_code.py

```diff
@@ -240,29 +241,35 @@ def show(code):
 @options.OVERWRITE()
 @options.SORT()
 @with_dbenv()
-def export(code, output_file, overwrite, sort):
+def export(code, output_file, overwrite, sort, **kwargs):
     """Export code to a yaml file."""
 
-    import yaml
+    other_args = {'sort': sort, **kwargs}
 
-    code_data = {}
-
-    for key in code.Model.model_fields.keys():
-        value = getattr(code, key).label if key == 'computer' else getattr(code, key)
-
-        # If the attribute is not set, for example ``with_mpi`` do not export it, because the YAML won't be valid for
-        # use in ``verdi code create`` since ``None`` is not a valid value on the CLI.
```

Comment:
I think it is good to keep this comment in the `_prepare_yml` implementation

Reply:
I actually tried that out and realized that it does work. When there's a `null` value in the resulting `yaml` file, upon `verdi code create`, the user gets prompted for that value when running interactively, while the default is set when running non-interactively. I still left the comment, but just modified it a bit.

Reply:
Did you try this explicitly for the `with_mpi` setting? Whether it works may depend on the option

Reply:
Yes, just did, and it worked. `verdi code test` went through on probs, and I also could run a `PwBaseWorkChain` with the created `Code`.